docs/install.rst

```diff
@@ -2,13 +2,6 @@
 Installation
 ############
 
-From PyPI
----------
-
-.. code-block:: bash
-
-    pip install -U aiogram
-
```

Comment:
Why you delete first? Delete second :) It's preferred way

Reply:
Had to add a section then